app/service/class-image-exchange.php

```diff
@@ -39,7 +39,7 @@ class Image_Exchange {
 
 	public function __construct() {
 
-		$this->api_url = 'http://images.fansided.test';
+		$this->api_url = 'https://images.fansided.com';
```

Comment:
I would consider adding a check to see if you're on a `.test` and hitting `image.fansided.test` instead. Means it will work locally when someone tries to debug it.

Reply:
I've been testing locally using the `images.fansided.com` url so this shouldn't be an issue for others when they test locally either.